### PR TITLE
[Task] POST Sprints #TDB-41

### PR DESCRIPTION
**Pull request by anjakammer**

### CODE REVIEW

File: app/controllers/sprints_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class SprintsController < ApplicationController
+  before_action :set_course, only: [:create]
+  before_action :set_sprint, only: [:update, :destroy]
+
+  # POST courses/course_id/sprints
```
Comment (ghost) [inline app/controllers/sprints_controller.rb]: missing a / here ;)

File: spec/requests/sprints_spec.rb [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      it 'returns status code 200' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'returns all course sprints' do
+        expect(json['sprints'].size).to eq(20)
+      end
+    end
+
+    context 'when course does not exist' do
```
Comment (ghost) [inline spec/requests/sprints_spec.rb:24]: aren't we already testing this in the course API tests?

**Review by ghost — APPROVED** [review]
two comments - but good to merge for me if you want

File: app/controllers/sprints_controller.rb (hunk 1 above)
Comment (anjakammer) [reply app/controllers/sprints_controller.rb]: :roll_eyes:

File: spec/requests/sprints_spec.rb (hunk 2 above)
Comment (anjakammer) [reply spec/requests/sprints_spec.rb:24]: The test here is checking the appearance of sprints inside a course.

**Pull request merged**